RED product updates: WO, WO Stats, FC, FC Perc, Tasseled Cap, Mangroves (#266)
GeoscienceAustralia/dea-knowledge-hub: docs/data/product/dea-mangrove-canopy-cover-landsat/_quality.md

## Accuracy

The following limitations reported for this product are relative to the *DEA Landsat Collection 2 version 2.0.2* of the mangroves dataset. The current DEA Mangroves version 4.0.0 product was produced with input datasets from DEA Landsat Collection 3. The similarities and differences between these two foundation dataset collections can be reviewed [here](https://www.dea.ga.gov.au/news/landsat-collection-upgrade). The primary impact on the collection 3 mangroves product is the change in pixel resolution from 25 to 30 m2 and use of the updated [collection 3 fractional cover](/data/product/dea-fractional-cover-landsat/) (FC) data product. Due to the increase in pixel spatial resolution, the reported accuracies and errors will be similar to those reported previously but not identical.

> During the upgrade process from collection 2 to collection 3, the mangrove thresholds for different classes were adjusted to more closely align with the Collection 2 version of the products. The current thresholds are set at (14, 40, 62).

Reviewer: ```suggestion
When this product was upgraded from Collection 2 to Collection 3, the mangrove thresholds for different classes were adjusted to maintain alignment with the Collection 2 version of the product. Therefore, the current mangrove thresholds are (14, 40, 62).
```
